yarn-project/end-to-end/src/sample-dapp/ci/index.test.mjs

```diff
@@ -10,6 +10,6 @@ describe('sample-dapp', () => {
   it('deploys and runs without errors', async () => {
     await waitForPXE(createPXEClient(PXE_URL));
     await deploy();
-    await main();
+    // await main();
```

Comment:
Was this intentional?

Reply:
Good catch, fixed in [4cd833a](https://github.com/AztecProtocol/aztec-packages/pull/3989/commits/4cd833af860814dcc5fe93f871797c1289209030)